tests/CLI/CLI.cs

```diff
@@ -1,9 +1,58 @@
 ﻿using CppSharp.AST;
+using CppSharp.AST.Extensions;
 using CppSharp.Generators;
+using CppSharp.Passes;
+using CppSharp.Types;
 using CppSharp.Utils;
 
 namespace CppSharp.Tests
 {
+    [TypeMap("IgnoredClassTemplateForEmployee")]
+    public class IgnoredClassTemplateForEmployeeMap : TypeMap
+    {
+        public override Type CLISignatureType(TypePrinterContext ctx)
+        {
+            return new CustomType("CLI::Employee^");
+        }
+
+        public override void CLIMarshalToManaged(MarshalContext ctx)
+        {
+            ctx.Return.Write($"gcnew CLI::Employee({ctx.ReturnVarName}.m_employee)");
+        }
+    }
+
+    public class CompleteIgnoredClassTemplateForEmployeeTypedefPass : TranslationUnitPass
```

Comment:
Can you explain the meaning of this pass? 

Is this to workaround any bug in the generator or just to force what you need from the test?

Reply:
Just to force what I need from the test.